target/linux/ath79/dts/qca9558_comfast_cf-wr650ac-v1.dts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9558_comfast_cf-wr650ac.dtsi"
+
+/ {
+        compatible = "comfast,cf-wr650ac-v1", "qca,qca9557";
+        model = "Comfast CF-WR650AC v1";
+};
+
+&spi {
+        status = "okay";
+        num-cs = <1>;
+
+        flash@0 {
+                compatible = "jedec,spi-nor";
+                reg = <0>;
+                spi-max-frequency = <25000000>;
+
+                partitions {
+                        compatible = "fixed-partitions";
+                        #address-cells = <1>;
+                        #size-cells = <1>;
+
+                        partition@0 {
+                                label = "u-boot";
+                                reg = <0x000000 0x020000>;
+                                read-only;
+                        };
+
+                        ART: partition@20000 {
```

Comment:
I've recently made an effort to harmonize art partition aliases and labels to be lower case: fcb920ffe7
It would be nice if you changed accordingly (ART => art)